x/staking/spec/03_messages.md

```diff
@@ -38,6 +38,8 @@ Additionally a self-delegation is made with the initial tokens delegation
 tokens `Delegation`. The validator always starts as unbonded but may be bonded
 in the first end-block.
 
+Thoughts: Seems to be nothing to do with MsgCreateValidator as it seems good.
```

Comment:
The MsgCreateValidator includes the validator's self-delegation.  So maybe we do want to buffer this, as it is still a delegation.